src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -2790,9 +2790,9 @@ TEST_F(EnsembleConfigurationValidationWithDemultiplexer, SuccessfulConfiguration
     ASSERT_NE(outputs.find(pipelineOutputName), outputs.end());
 
     const auto& input_A = inputs.at(pipelineInputName);
-    EXPECT_EQ(input_A->getEffectiveShape(), shape_t({1, 10}));
+    EXPECT_EQ(input_A->getShape_3(), Shape({1, 10}));
```

Comment:
You can do:
 `EXPECT_EQ(input_A->getShape_3(), Shape{1, 10});`

Reply:
I can't since compiler then can't know which constructor I want to call:
```error: macro "EXPECT_EQ" passed 3 arguments, but takes just 2```
Its enforced by Gtest unfortunately. I could initialize shape in different line but that would miss the point.